docs/2.getting-started/2.getting-started.mdx

```diff
@@ -11,14 +11,14 @@ import { PrevButton } from '../components/prevButton.tsx'
 Let's create our first Botonic bot!
 
 ## Step 1 – Install the Botonic Command Line Interface
-1. Close any terminals/command prompts you may have open.
-2. Install the latest  LTS version of [NodeJS installer](https://nodejs.org/). You can also install Yarn.
-3. Verify the installation version by running `npm --version` and `node --version`
-4. Perform a global installation by running  `sudo npm install -g @botonic/cli` (Mac/Linux) or `npm install -g @botonic/cli` (as admin on Windows).
-5. Wait until the CLI is installed.
+1. Install the latest  LTS version of [NodeJS installer](https://nodejs.org/). You can also install Yarn.
```

Comment:
@marcrabat But maybe we don't support the latest LTS, right? I would specify here the major version that we have tested

Reply:
Version specified